problems/sevenBoom.js

```diff
@@ -4,8 +4,17 @@
 * @param {number} n - The number to count up to
 * @returns {number[]} - An array matching the pattern described above
 */
-function sevenBoom() {
-
+const sevenBoom = n => {
+    let arr = []
+    for (let i = 1; i <= n; i++) {
+        if (i % 7 === 0 || i % 10 === 7) {
```

Comment:
This may pass the tests but doesn't take into consideration numbers like 75 which include a 7 but doesn't end in 7 or go into 7 evenly